action.yaml

```diff
@@ -2,17 +2,25 @@ name: 'Aqua Security Trivy'
 description: 'Scans container images for vulnerabilities with Trivy'
 author: 'Aqua Security'
 inputs:
+  scan-type:
+    description: 'Scan type to use for scanning vulnerability'
+    required: false
+    default: 'image'
   image-ref:
-    description: 'image reference'
+    description: 'image reference(for backward compatibility.)'
     required: true
+  scan-ref:
+    description: 'Scan reference'
+    required: false
+    default: ''
```

Comment:
I see that you use `/github/workspace` as the value for this option here https://github.com/rahul2393/trivy-ci-test/commit/d5be8c31c2ed29f6e3dec4acd8e0e0772b1cfca7#diff-246cd0c2f7db532638dd80a92ac011f49b3d26038983a4c0169ea8f8a5c39280R15

I think most users won't know about this in order to use the action. We should add logic somehow that in the case of `scan-type` as `fs` we default to using `/github/workspace/` as the value.

Reply:
@simar7 default workspace for docker is `github/workspace` so it will work if user won't define scan-ref with scan-type=fs
https://github.com/rahul2393/trivy-ci-test/commit/e69e72747b1d6d8e17a0e97248d7443234383ec9